x-pack/test/functional_execution_context/tests/browser.ts

```diff
@@ -328,18 +328,18 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
           description: 'execution context propagates to Kibana logs',
           predicate: (record) =>
             isExecutionContextLog(record?.message, {
-              parent: {
-                type: 'application',
-                name: 'dashboard',
-                id: '7adfa750-4c81-11e8-b3d7-01146121b73d',
-                description: '[Flights] Global Flight Dashboard',
-                url: '/view/7adfa750-4c81-11e8-b3d7-01146121b73d',
+              type: 'application',
```

Comment:
Have you run these tests? They seem to be skipped due to flakiness.  But you can un-skip locally to check they are green.